extensions/dart/languages/dart/tasks.json

```diff
@@ -0,0 +1,32 @@
+[
+  {
+    "label": "dart run",
+    "command": "dart",
+    "args": ["run"],
+    "tags": ["dart-main"]
```

Comment:
None of your queries set a `dart-main` tag, which means that this runnable is never going to be created I think

Reply:
I have removed this for now, could not find a way to detect 'dart' app. A 'dart main' is where we don't import flutter or the test packages.